samples/snippets/src/main/java/com/example/spanner/QueryProtoColumnSample.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.spanner;
+
+import com.example.spanner.SingerProto.Genre;
+import com.example.spanner.SingerProto.SingerInfo;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+
+public class QueryProtoColumnSample {
+
+  static void queryProtoColumn() {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project";
+    String instanceId = "my-instance";
+    String databaseId = "my-database";
+
+    try (Spanner spanner =
+        SpannerOptions.newBuilder().setProjectId(projectId).build().getService()) {
+      DatabaseClient client =
+          spanner.getDatabaseClient(DatabaseId.of(projectId, instanceId, databaseId));
+      queryProtoColumn(client);
+    }
+  }
+
+  /**
+   * Method to query Singer Table using DQL. Assuming a table Singer with columns (singer_id,
+   * singer_info, genre, singer_info_list, genre_list)
```

Comment:
Can we add the DDL of the Singer table as a comment here?
This would make the column type of fields clear. Currently it is confusing on what each field contains.